### PR TITLE
add routeHost as new parameter in cr

### PR DESCRIPTION
**Pull request by ZhuoxiLi**
I add routHost in `api/v3/commonservice_types.go` & `config/crd/bases/operator.ibm.com_commonservices.yaml`. 

Also create `controllers/routeHost/routeHost.go`, so the value of routeHost we assigned in cs cr will be in ibm-management-ingress-operator's cr. 
~~^^ correct me if I am wrong. I am working on adding the logical in `controllers/render_template.go`~~ Done

### CODE REVIEW

File: bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -43,17 +43,9 @@ spec:
   customresourcedefinitions:
     owned:
       - description: CommonService is the Schema for the commonservices API
-        displayName: CommonService
+        displayName: Common Service
         kind: CommonService
         name: commonservices.operator.ibm.com
-        specDescriptors:
```
Comment (horis233) [inline bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:49]: @ZhuoxiLi Please revert changes in the CSV

File: bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -327,5 +327,5 @@ spec:
   maturity: alpha
   provider:
     name: IBM
-  replaces: ibm-common-service-operator.v3.7.3
+  replaces: ibm-common-service-operator.v3.8.0
```
Comment (ZhuoxiLi) [inline bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:330]: Do we want to update the version to v3.8.0?

Comment (ZhuoxiLi) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml]: sure, update to 3.7.4

**Review by horis233 — APPROVED** [review]

**Pull request merged**